websites/Y/YouTube/video_sources/movies.ts

```diff
@@ -0,0 +1,28 @@
+import { Resolver } from "../util";
+
+function isActive(): boolean {
+	return true;
```

Comment:
```suggestion
	return !!getTitle() && !!getUploader();
```
Should it not be this?